packages/mongo/package.js

```diff
@@ -34,7 +34,8 @@ Package.onUse(function (api) {
     'diff-sequence',
     'mongo-id',
     'check',
-    'ecmascript'
+    'ecmascript',
+    'mongo-dev-server',
```

Comment:
Relating to your comment:

> I'm thinking we could remove mongo-dev-server as a mongo dependency and use the upgrades thing to just add this package to all current meteor projects on upgrade and add it to the default packages when creating a new project.

I like this idea - Not forcing `mongo-dev-server` as a dependency, and using an `upgrader` instead, gets my vote!